src/styles/main.scss

```diff
@@ -1,3 +1,63 @@
+$height: 100px;
+$width: 100px;
+$padding: 10px;
+$gap: 1px;
+
 body {
   margin: 0;
+  padding: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: #{$gap};
+  padding: #{$padding};
+  width: $width * 7 + $gap * 6;
+
+  &__day {
+    height: #{$height};
+    width: #{$width};
+    background-color: #eee;
+    border: 1px solid black;
+    box-sizing: border-box;
+    position: relative;
+
+    &:hover {
+      cursor: pointer;
+      background-color: pink;
+      transition: transform 0.5s;
```

Comment:
Move this CSS prop to calendar__day selector as transform should be smooth to both states, when the mouse enters and when it leaves.